libs/community/tests/integration_tests/retrievers/document_compressors/test_embeddings_filter.py

```diff
@@ -1,5 +1,6 @@
 """Integration test for embedding-based relevant doc filtering."""
 import numpy as np
+import pytest
```

Comment:
```suggestion
```

Reply:
if I delete this annotation, the test is not run

Reply:
Make sure your environment is up to date. We use a plugin to automatically run asyncio code. 

See:
https://github.com/langchain-ai/langchain/blob/5ba1899cd72dd57ee760d63137e2c92731c7f7a4/libs/langchain/pyproject.toml#L145-L145

Reply:
https://pytest-asyncio.readthedocs.io/en/latest/concepts.html#auto-mode